specifyweb/frontend/js_src/lib/components/Header/index.tsx

```diff
@@ -107,30 +108,28 @@ export function Header({
     >
       <h1 className="contents">
         <a
-          className={`
-              flex items-center
+          className={`flex
+              items-center
               ${isCollapsed ? 'p-2' : 'p-4'}
             `}
           href="/specify/"
         >
           {/* Both logs are loaded to prevent flickering on collapse/expand */}
-          <img
-            alt=""
-            className={`
-                hover:animate-hue-rotate
+          <span
+            className={`w-full
```

Comment:
you removed `alt=""` - it was important
read online the difference between having empty alt and no alt